File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        d = {};
        if len(s) != len(t):
            return False
        for i in s:
            d[i] = d.get(i, 0) + 1
        for i in t:
            if i not in d or d[i] <= 0:
                return False
            else:
                d[i] = d[i] - 1
        return True

File: Data Structures & Algorithms/reverse-bits/submission-1.cs
public class Solution {
    public uint ReverseBits(uint n) {
        uint res = 0;
        for (int i = 0; i < 32; i++)
        {
            if ((n & (1u << i)) != 0)
            {
                res = res | (1u << (31-i));
            }
        }
        return res;
    }
}
